cmd/mr_checkout_test.go

```diff
@@ -88,6 +89,42 @@ func Test_mrCheckoutCmd_track(t *testing.T) {
 	}
 	remotes := string(gitOut)
 	require.Contains(t, remotes, "zaquestion	git@gitlab.com:zaquestion/test.git")
+
+	// confirm that new branch tracks zaquestion/mrtest
+	cmd = exec.Command("git", "rev-parse", "--symbolic-full-name", "--abbrev-ref", "mrtest_track@{upstream}")
+	cmd.Dir = repo
+	gitOut, err = cmd.CombinedOutput()
+	if err != nil {
+		t.Fatal(err)
+	}
+	require.Equal(t, "zaquestion/mrtest", strings.TrimSpace(string(gitOut)))
```

Comment:
This assertion is meant to confirm that the new branch is set to track the upstream `zaquestion/mrtest`. My understanding is that the existing test looks for various circumstantial signs that this has been set up correctly, but doesn't actually confirm the specific upstream relation. Happy to revert this if you feel it's not necessary. Thanks!